src/analysis/inner_product_space/adjoint.lean

```diff
@@ -16,6 +16,9 @@ Given an operator `A : E →L[𝕜] F`, where `E` and `F` are Hilbert spaces, it
 We then use this to put a C⋆-algebra structure on `E →L[𝕜] E` with the adjoint as the star
 operation.
 
+This construction is used to define an adjoint for linear maps (i.e. not continuous) between
+finite dimensional spaces.
```

Comment:
In the module docstring can you, within the sentences you have, mention the name of the constructions?  Eg "is used to define `linear_map.adjoint`, an adjoint for linear maps ...." and similar for `continuous_linear_map.adjoint`.

Reply:
Instead of that, I just added a "Main definitions" section in the module docstring.